src/Promitor.Scraper.Tests.Unit/Serialization/v1/Providers/ResourceDeserializerTestBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Promitor.Core.Scraping.Configuration.Serialization;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
+using Xunit;
+
+namespace Promitor.Scraper.Tests.Unit.Serialization.v1.Providers
+{
+    public abstract class ResourceDeserializerTestBase
```

Comment:
Can we leave out the base suffix?
```suggestion
    public abstract class ResourceDeserializerTest
```

It's a recommendation on to use it https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/base-classes-for-implementing-abstractions

Reply:
Sure, happy to, although if you read the guidelines they say not to include the "Base" suffix when the class is exposed via a public API, which isn't the case here because it's a test class.

Reply:
That is correct but I prefer to have the same naming convention regardless of how it is exposed, it's a bit more consistent - Not a blocker though

Reply:
No worries - I've renamed it.

I wasn't meaning that I didn't want to rename it, I just had taken your comment a bit too literally.

Reply:
No worries, I was just explaining why I prefer it that way